blocks/example-file-block/index.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+import { FileBlockProps } from "@githubnext/blocks";
+
+// seems like it's not possible to define the props all at once from `FileBlockProps`
+// see https://github.com/vuejs/core/issues/4294
+const { content } = defineProps<{
+  content: FileBlockProps["content"];
+}>();
```

Comment:
This is a bummer. Not worthing blocking this PR on finding a solution here, but we should file a ticket to come back and investigate.

Reply:
yeah super bummer. The ticket doesn't make me hopeful that there's a straightforward solution.